Cargo.toml

```diff
@@ -11,6 +11,12 @@ inherits = "dev"
 debug = 0
 incremental = false
 
+[profile.release]
```

Comment:
Given that the majority of savings comes from `strip`, I decided to leave `opt-level` to the default (which is `3` for release) instead of `"s"` which would've shaved another ~800kb (16%) off `rad-gov`. Thus our binary is still optimized for performance.